go.mod

```diff
@@ -7,6 +7,8 @@ require (
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/sirupsen/logrus v1.7.0
 	github.com/smartystreets/goconvey v1.6.4
+	github.com/solarwinds/snap-plugin-lib/v2 v2.0.2 // indirect
```

Comment:
this is strange? reference to the itself?

Reply:
good catch... I've run go mod update and tidy and it's gone now